### ISSUE TITLE
Add-on crash, Blender 2.93.9

### ISSUE THREAD

**Issue by theffect** [s-leger/blender_cad_transforms#82]
Blender 2.93.9 and even 2.93.2
Start a new "General"
Add a mesh
Switch to CAD transforms
Type G for measuring and:
```
Python: Traceback (most recent call last):
  File "/home/<user>/.config/blender/2.93/scripts/addons/slcad_transform/slcad_transform.py", line 3192, in invoke
    self.init(context)
  File "/home/<user>/.config/blender/2.93/scripts/addons/slcad_transform/slcad_transform.py", line 3105, in init
    slcadsnap.start(context, prefs.snap_to_loose)
  File "/home/<user>/.config/blender/2.93/scripts/addons/slcad_transform/slcad_snap.py", line 223, in start
    self.update_gl_snap()
  File "/home/<user>/.config/blender/2.93/scripts/addons/slcad_transform/slcad_snap.py", line 697, in update_gl_snap
    self.gl_stack.set_snap_mode(self.snap_mode)
  File "/home/<user>/.config/blender/2.93/scripts/addons/slcad_transform/snap_context/__init__.py", line 235, in set_snap_mode
    self.update_all()
  File "/home/<user>/.config/blender/2.93/scripts/addons/slcad_transform/snap_context/__init__.py", line 223, in update_all
    self._offscreen.texture_color.clear(format='FLOAT', value=(0.0, 0.0, 0.0, 0.0))
AttributeError: 'GPUOffScreen' object has no attribute 'texture_color'

location: <unknown location>:-1
```

**Comment by theffect** [comment]
Probably related to: https://developer.blender.org/T84227
Fixed at Blender's LTS Release 2.93.6

**Comment by theffect** [comment]
duplicate of #62
